src/main/service/listManipulation/magsStandardListService.ts

```diff
@@ -0,0 +1,92 @@
+import moment from 'moment-timezone';
+import {DataManipulationService} from '../dataManipulationService';
+import {DateTimeHelper} from '../../helpers/dateTimeHelper';
+
+const dataManipulationService = new DataManipulationService();
+const dateTimeHelper = new DateTimeHelper();
+
+export class MagsStandardListService {
+  public manipulatedMagsStandardListData(magsStandardListData: object, language: string, languageFile: string): object {
+    let hearingCount = 0;
+    magsStandardListData['courtLists'].forEach(courtList => {
+      courtList['courtHouse']['courtRoom'].forEach(courtRoom => {
+        courtRoom['session'].forEach(session => {
+          session['sittings'].forEach(sitting => {
+            const allHearings = [];
+            session['formattedJudiciaries'] = dataManipulationService.findAndManipulateJudiciary(sitting);
+            this.formatCaseTime(sitting, 'h:mma');
+            hearingCount = hearingCount + sitting['hearing'].length;
+            sitting['hearing'].forEach(hearing => {
+              sitting['formattedDuration'] = dateTimeHelper.formatDuration(sitting['durationAsDays'] as number,
+                sitting['durationAsHours'] as number, sitting['durationAsMinutes'] as number, language, languageFile);
+              hearing.party.forEach(party => {
+                const hearingString = JSON.stringify(hearing);
+                const hearingObject = JSON.parse(hearingString);
+                if (hearingObject?.party) {
```

Comment:
Hhmm why do we need the if statement here? It is within the forEach for party so surely it has party?

Reply:
fixed.